### DISCUSSION TITLE
"Need and LDAP Maintainer?"

### DISCUSSION THREAD

**Discussion by chris2fr** [docker-mailserver/docker-mailserver#2593] [General]
Are these to be my last words?  

"Need and LDAP Maintainer?"

I can do that. 

( b e   h a p p y   t o )

**Comment by polarathene** [comment]
We would appreciate contributions for LDAP!

You can help with pull requests to fix LDAP issues, or improving our docs for LDAP users. Some issues about LDAP we're unable to help users due to a lack of LDAP knowledge.
